web/profiles/joinup/config/install/geocoder.geocoder_provider.webtools_geocoding.yml

```diff
@@ -6,4 +6,5 @@ dependencies:
 id: webtools_geocoding
```

Comment:
This config is now a "default configuration" (even, is the same as the sync version). We should move it in the module where it belongs as functionality. Probably, `joinup_event`. Let's start to cleanup of the profile :)